vulnerabilities/improvers/default.py

```diff
@@ -39,21 +39,22 @@ def interesting_advisories(self) -> QuerySet:
         return Advisory.objects.all()
 
     def get_inferences(self, advisory_data: AdvisoryData) -> Iterable[Inference]:
-
         if not advisory_data:
             return []
 
         if advisory_data.affected_packages:
             for affected_package in advisory_data.affected_packages:
-                affected_purls, fixed_purl = get_exact_purls(affected_package)
-                yield Inference(
-                    aliases=advisory_data.aliases,
-                    confidence=MAX_CONFIDENCE,
-                    summary=advisory_data.summary,
-                    affected_purls=affected_purls,
-                    fixed_purl=fixed_purl,
-                    references=advisory_data.references,
-                )
+                # To deal with multiple fixed versions in a single affected package
```

Comment:
What is the comment meaning?

Reply:
I don't know -- this change was made by @TG1999 .  @TG1999 -- can you please respond to @pombredanne 's question?  If there's more we could add to this comment, please let me know and I'll make the change.

Reply:
@pombredanne the previous improver design was just to process a single fixed version over multiple affected packages, but now we can store multiple fixed versions in a single range(because of invert functionality) so we need to change the way we process the advisory data from now.